### DISCUSSION TITLE
Cut and paste graphs

### DISCUSSION THREAD

**Discussion by sstebbinspeacecorps** [hodcroftlab/covariants#98] [Ideas]
Hi - is there an easy way to cut and paste an individual graph? I'd like to highlight in powerpoint. Thanks!

**Comment by emmahodcroft** [comment]
Hi @sstebbinspeacecorps , thanks for reaching out! The best thing I can suggest is to use a screen grab. Many operating systems have programs that let you select just part of the screen these days, you can google to find out the instructions for your specific OS. I hope that helps!

**Comment by sstebbinspeacecorps** [comment]
Thanks, and thanks for CoVariants. It is an awesome website, and I expect will get even better as more countries get better at reporting more genetic analyses.

**Reply by emmahodcroft** [reply]
Thank you @sstebbinspeacecorps for the kind words! 🙏 

Just so you know - in your message above, I guess it didn't trim the email properly, but your email address is visible. You might want to edit it to remove this before bots find it! 😁

**Comment by sstebbinspeacecorps** [comment]
Hmmm…good thought.

**Reply by emmahodcroft** [reply]
Just so you know Steve, I've gone ahead and edited your comments to remove your phone numbers & email (and the general email stuff), just for your privacy!
I wouldn't normally edit the comments but I don't want your personal info lingering around too long publicly :)

**Reply by ivan-aksamentov** [reply]
Just to clarify Emma's messages. 

All these messages are visible on GitHub here:
```
https://github.com/hodcroftlab/covariants/discussions/98
```

When you create a Discusion thread or an Issue thread, replies in the thread are also duplicated as emails, and if you reply to that email, that got posted in the thread too, along with the signature etc.
Emma edited the messages to remove signatures with personal details. 

This is very confusing and I don't know why Github does that.

**Comment by sstebbinspeacecorps** [comment]
Many thanks.

**Comment by sstebbinspeacecorps** [comment]
Hi Emma – thanks again for your previous help in keeping my personal/work information off the public site?

Do you know what has happened to South Africa’s COVID-19 Variants data? Nothing has been reported in several months.

Thanks!



Sam

**Reply by emmahodcroft** [reply]
Hi Sam, I'm afraid I don't! I know countries can have backlogs of sequencing or sometimes run into trouble with uploading, but here I'm afraid that's all speculation and I don't know any further info. Sorry!

**Comment by sstebbinspeacecorps** [comment]
Many thanks. I so much enjoy utilizing the CoVariants website and information. Best wishes!
